src/gui_common/thrive_theme.tres

```diff
@@ -1,32 +1,18 @@
-[gd_resource type="Theme" load_steps=73 format=2]
+[gd_resource type="Theme" load_steps=59 format=2]
 
 [ext_resource path="res://src/gui_common/fonts/Jura-Regular-AlmostSmaller.tres" type="DynamicFont" id=1]
-[ext_resource path="res://assets/textures/gui/bevel/grab.png" type="Texture" id=2]
```

Comment:
Looks like Godot automatically entirely wrecked this file so you'll want to revert all changes in here.